python/Produce_Config_File.py

```diff
@@ -0,0 +1,131 @@
+		ret_int='kGreen'
+	elif j % 7 == 6:
+		ret_int='kGreen+2'
+	
+
+	return ret_int
+
+
+				
+
+args = parser.parse_args()
+
+
+
+
+
+text_file = open(str(args.OutputName)+".cfg", "w")
+j=0
+
+text_file.write("[BEGIN_CANVAS]\n")
+text_file.write("    Canv_Axis_NDiv = '"+str(args.AxisNDiv)+"'; #X,Y\n")
+text_file.write("    Canv_Dim= '"+str(args.CanvDim)+"'; #X,Y\n")
+text_file.write("    Canv_DrawOpt = '"+str(args.CanvDrawOpt)+"';\n")
+text_file.write("    Canv_Grid_XY = '"+str(args.CanvGridXY)+"';\n")
+text_file.write("    Canv_Latex_Line = '"+str(args.LatexLine)+"';\n")
+text_file.write("    Canv_Latex_Line = '"+str(args.LatexLine1)+"';\n")
+text_file.write("    Canv_Latex_Line = '"+str(args.LatexLine2)+"';\n")
+text_file.write("    Canv_Legend_Dim_X = '"+str(args.CanvLegDimX)+"';\n")
+text_file.write("    Canv_Legend_Dim_Y = '"+str(args.CanvLegDimY)+"';\n")
+text_file.write("    Canv_Legend_Draw = '"+str(args.CanvLegDraw)+"'; \n")
+text_file.write("    Canv_Log_XY = '"+str(args.CanvLogXY)+"';\n")
+text_file.write("    Canv_Logo_Pos = '"+str(args.CanvLogoPos)+"'; #0, 11, 22, 33\n")
+text_file.write("    Canv_Logo_Prelim = '"+str(args.CanvLogoPrelim)+"';\n")
+text_file.write("    Canv_Margin_Top = '"+str(args.CanvMarginTop)+"';\n")
+text_file.write("    Canv_Margin_Bot = '"+str(args.CanvMarginBot)+"';\n")
+text_file.write("    Canv_Margin_Lf = '"+str(args.CanvMarginLf)+"';\n")
+text_file.write("    Canv_Margin_Rt = '"+str(args.CanvMarginRt)+"';\n")
+text_file.write("    Canv_Name = '"+str(args.CanvName)+"';\n")
+text_file.write("    Canv_Plot_Type = '"+str(args.CanvPlotType)+"';\n")
+text_file.write("    Canv_Range_X = '"+str(args.CanvRangeX)+"'; #X1, X2\n")
+text_file.write("    Canv_Range_Y = '"+str(args.CanvRangeY)+"'; #Y1, Y2\n")
+text_file.write("    Canv_Title_Offset_X = '"+str(args.CanvTitleOffsetX)+"';\n")
+text_file.write("    Canv_Title_Offset_Y = '"+str(args.CanvTitleOffsetX)+"';\n")
+text_file.write("    Canv_Title_X = '"+str(args.CanvTitleX)+"';\n")
+text_file.write("    Canv_Title_Y = '"+str(args.CanvTitleY)+"';\n")
+for f in args.file:
+	
+	workbook = xlrd.open_workbook(str(f))
+	ws = workbook.sheet_by_index(int(args.SelectSheetNum))
+	Gcharacter=int(f.index('G'))
+	legEntry=f[int(Gcharacter):int(Gcharacter+18)]
+	
+	Marker_Style = int(20+j)
+	text_file.write("    [BEGIN_PLOT]\n" )
+	text_file.write("        Plot_Color = '"+str(Color(j))+ "';\n" )
+	text_file.write("        Plot_LegEntry = '"+str(legEntry)+"';\n" )
+	text_file.write("        Plot_Line_Size = '1';\n" )
+	text_file.write("        Plot_Line_Style = '1';\n" )
+	text_file.write("        Plot_Marker_Size = '1';\n" )
+	text_file.write("        Plot_Marker_Style = '"+str(Marker_Style)+"';\n" )
+	text_file.write("        Plot_Name = '"+str(legEntry)+"'; \n" )
+	text_file.write("        [BEGIN_DATA]\n" )
+	text_file.write("          VAR_INDEP,VAR_DEP,VAR_INDEP_ERR,VAR_DEP_ERR\n" )
+	j=j+1
+
+	for i in range(int(args.SelectRowStart),int(args.SelectRowEnd)):
```

Comment:
use a more descriptive variable than `i`

In options.py you defined the type of `SelectRowStart` and `SelectRowEnd` to be `int`.   You do not need to cast it again as an `int().` To be changed.